### DISCUSSION TITLE
[Question] Hard interact with icons on the rear of desktop

### DISCUSSION THREAD

**Discussion by neronguyen** [prasanthrangan/hyprdots#1043] [Q&A]
I use android studio and feel so hard to click on menu icons on the left, how to config so the system receive less coordinates to those event ?

![240309_18h11m37s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/50225603/23fac2b9-7f82-4b8e-81a1-0840f13b5c68)

**Comment by kRHYME7** [comment]
Try to toggle it to float.

**Reply by neronguyen** [reply]
Sorry what you mean with toggle it ?
